### PR TITLE
React Router: `NavigateFunction` signatures are made callable

### PR DESCRIPTION
**Pull request by sgrishchenko**
cc @aerialist7

### CODE REVIEW

File: kotlin-react-router/src/jsMain/generated/react/router/NavigateFunction.kt [hunk 1]
```diff
@@ -1,5 +1,8 @@
+@file:JsModule("react-router")
```
Comment (turansky) [inline kotlin-react-router/src/jsMain/generated/react/router/NavigateFunction.kt:1]: Module is for `external` only

File: kotlin-react-router/src/jsMain/generated/react/router/NavigateFunction.kt [hunk 2]
```diff
@@ -11,15 +14,13 @@ import remix.run.router.To
...
 sealed class NavigateFunction {
-    inline operator fun invoke(to: To) {
-        asDynamic()(to)
-    }
 
-    inline operator fun invoke(to: To, options: NavigateOptions) {
-        asDynamic()(to, options)
-    }
+    @Suppress("NOTHING_TO_INLINE")
+    inline operator fun invoke(to: To, options: NavigateOptions): Unit = asDynamic()(to, options)
```
Comment (turansky) [inline kotlin-react-router/src/jsMain/generated/react/router/NavigateFunction.kt:19]: Separate `invoke(To)` signature required

File: kotlin-react-router/src/jsMain/kotlin/react/router/NavigateFunction.ext.kt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package react.router
+
+import remix.run.router.To
+
+@Suppress("NOTHING_TO_INLINE")
+inline operator fun NavigateFunction.invoke(to: To): Unit = asDynamic()(to)
```
Comment (turansky) [inline kotlin-react-router/src/jsMain/kotlin/react/router/NavigateFunction.ext.kt:6]: Same priority for calls required

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react-router/src/jsMain/generated/react/router/NavigateFunction.kt (hunk 1 above)
Comment (sgrishchenko) [reply kotlin-react-router/src/jsMain/generated/react/router/NavigateFunction.kt:1]: I know, this fix is in the road map of `karakum` converter

**Pull request merged**